elastic/kibana
# [Entity Analytics] Add criticality column to All Users table In Explore pages

This PR, a follow-up to #186375, adds a new column for Asset Criticality to the "All hosts" tab in the Explore/Hosts page.
If any of the hosts has criticality data assigned, it will be displayed in the new column. If no criticality has been assigned, the field is left blank.

<img width="1530" alt="Screenshot 2024-06-19 at 13 46 52" src="https://github.com/elastic/kibana/assets/2423976/054d0bb0-8858-40ae-ab20-7fe6f31e5858">

### How to test

1. Make sure you have test data with Asset Criticality.
    - Either use the [datagen tool](https://github.com/elastic/security-documents-generator) or just make sure to assign criticality to already existing hosts
2. Enable Asset Criticality settings in `Stack Management > Kibana > Advanced Settings`
3. Navigate to `Explore > Users > All Users`
